R/msnset2se.R

```diff
@@ -0,0 +1,48 @@
+#' SummarizedExperiment to MSnSet object conversion
+#'
+#' \code{se2msnset} generates an MSnSet object from an SummarizedExperiment object.
+#'
+#' @param se SummarizedExperiment,
+#' Object which will be turned into an MSnSet object.
+#' @return MSnSet object.
+#' @export
+se2msnset <- function(se) {
+  # Check input
+  assertthat::assert_that(inherits(se, "SummarizedExperiment"))
+
+  # Extract expression, feature and pheno data
+  raw <- SummarizedExperiment::assay(se)
```

Comment:
Currently we also have no dependency on `SummarizedExperiment`. That means you have to add `SummarizedExperiment` to the `DESCRIPTION` file. I would suggest to add it into `Suggests:` and use `if (requireNamespace("SummarizedExperiment"))` here to check whether it is available or not.

Reply:
The `::` should not necessary. You only need it if you have two functions with the same name in two imported packages. Beside adding `SummarizedExperiment` to the `DESCRIPTION` file you have to import the class and the methods you use via `NAMESPACE`: `importClassesFrom(SummarizedExperiment, SummarizedExperiment)` and `importFrom(SummarizedExperiment, assay, colData, rowData)`.